src/core/control/settings/SettingsEnums.h

```diff
@@ -77,6 +77,20 @@ enum IconTheme {
     ICON_THEME_LUCIDE = 1,
 };
 
+/**
+ * The user-selectable Page Preview Decoration style 
+ */
+enum SidebarNumberingStyle {
```

Comment:
nit: I would change this to `enum class` so that the entries are scoped under `SidebarNumberingStyle`, e.g.
```c++
enum class SidebarNumberingStyle {
    NONE = 0,
    // ...
}

// use it:
void foo() {
    auto myStyle = SidebarNumberingStyle::NONE;
}
```